todo-frontend-angular/src/app/todo/todolist/todolist.component.ts

```diff
@@ -1,15 +1,16 @@
 import { Component, OnInit } from '@angular/core';
+import { TodoService } from '../todo.service';
 
 @Component({
   selector: 'app-todolist',
   templateUrl: './todolist.component.html',
   styleUrls: ['./todolist.component.scss']
 })
 export class TodolistComponent implements OnInit {
-
-  constructor() { }
+  todos: any;
+  constructor(private readonly todoService: TodoService) { }
 
   ngOnInit(): void {
+    this.todoService.getTodos().subscribe(response=>{this.todos=response ; console.log(this.todos)})
```

Comment:
Good subscribe! Rxjs is deprecating this way of subscribing though, so instead do: `subscribe({next: () => {}})`